ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -182,6 +184,34 @@ protected boolean setupFillPaint(Paint paint, float opacity) {
               (int) (mFillColor[2] * 255),
               (int) (mFillColor[3] * 255));
           break;
+        case 1: // linear gradient
```

Comment:
Why is `1` a linear gradient? Can you use some constant instead or add a comment with link to docs explaining why `1` should be a linear gradient? This is very non-obvious.